qemu/tests/ksm_overcommit.py

```diff
@@ -631,11 +615,6 @@ def split_parallel():
                        "living VM" % lvms[i].name)
 
         lsessions.append(lvms[i].wait_for_login(timeout=360))
-        try:
-            tmp = open(params.get('pid_' + vm_name), 'r')
-            params['pid_' + vm_name] = int(tmp.readline())
-        except Exception:
-            test.fail("Could not get PID of %s" % (vm_name))
 
```

Comment:
Checking if QEMU process PID be stored the pid file is not the check point for this case ?

Reply:
In fact, i am ok with the change that remove pidfile,  i do not think this file makes mush sense.  But a check point be removed with this change.  If no meaning for ksm test, i am ok.

Reply:
It has nothing to do with ksm feature. So no need to check that. Thanks.